Drop-In/src/test/java/com/braintreepayments/api/dropin/AddCardActivityUnitTest.java

```diff
@@ -546,6 +547,21 @@ public void onPaymentMethodNonceCreated_sendsAnalyticsEvent() throws JSONExcepti
         verify(mActivity.braintreeFragment).sendAnalyticsEvent("sdk.exit.success");
     }
 
+    @Test
+    public void nonFormFieldError_callsFinishWithError() {
+        setup(mock(BraintreeFragment.class));
+
+        ErrorWithResponse error = new ErrorWithResponse(422, stringFromFixture("responses/three_d_secure/authentication_response_with_error.json"));
```

Comment:
Why is this JSON specific to 3DS authentication?

Reply:
If it is specific to 3ds, perhaps the test name should reflect it.

Reply:
The code itself isn't specific to 3D Secure, so the test name reflects that.  However, the case that prompted the bug fix did have to do with 3D Secure, which is the reason for the name of the fixture.

However, the way the code is written, an error could come from a non 3D Secure case and flow through this code path as well, so we did not want to give the impression that _only_ 3DS errors would make it here.

Reply:
Curious if the fixture should uphold for a more generic error case as well? Or if we should add another.

Reply:
We opted to rewrite the test to be very specific in testing the error payload, and avoid any unnecessary 3D Secure details in the JSON blob.

Reply:
This looks good to me! 👍